### PR TITLE
Added Jenkinsfile

### PR DESCRIPTION
**Pull request by oshercc**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+              when {
+                  branch 'master'
+              }
+
+              steps {
+                  withCredentials([usernamePassword(credentialsId: 'ocpmetal_cred', passwordVariable: 'PASS', usernameVariable: 'USER')]) {
+                      sh '''docker login quay.io -u $USER -p $PASS'''
+                  }
+
+                  sh '''docker tag  ${INSTALLER} ${INSTALLER}:latest'''
```
Comment (romfreiman) [inline Jenkinsfile:34]: what is installer?

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,41 @@
+pipeline {
+
+  environment {
+        AGENT_IMAGE = 'quay.io/introspector-build'
```
Comment (romfreiman) [inline Jenkinsfile:4]: how can it be that agent image is build?

File: Jenkinsfile (hunk 1 above)
Comment (filanov) [reply Jenkinsfile:34]: We need to make sure that we use the same image that we use for tests. so maybe you should create and env for it.
Take a look how it's done in assisted-service https://github.com/openshift/assisted-service/blob/master/Jenkinsfile#L4

File: Jenkinsfile (hunk 2 above)
Comment (romfreiman) [reply Jenkinsfile:4]: @oshercc ?

**Pull request closed without merging**